internal/interfaces/grpc/interceptor/interceptor.go

```diff
@@ -10,6 +11,7 @@ import (
 func UnaryInterceptor(svc *macaroons.Service) grpc.ServerOption {
 	return grpc.UnaryInterceptor(
 		middleware.ChainUnaryServer(
+			grpc_recovery.UnaryServerInterceptor(),
```

Comment:
What's this for?

Reply:
to recover for panic and return the error to the grpc response I guess?

Reply:
yes, before this panic would make daemon crash